.github/workflows/deploy_cms.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
         uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076 # v2.0.1
 
       - name: Build Push and Tag
+        continue-on-error: true # Set to true for manual deployment in case of rollback because this step would fail if the image is already present in the ECR
```

Comment:
```suggestion
        # Enabling the "continue on error" option allows for a manual rollback
        # to be performed in case of any issues. Without this option, the step
        # will fail if the image already exists in the Elastic Container
        # Registry (ECR). However, by activating this option, the deployment
        # process will proceed to the next steps even if the ECR image already
        # exists
        continue-on-error: true
```